app/controllers/conferences_controller.rb

```diff
@@ -33,8 +32,15 @@ def redirect
 
   private
 
-  def generate_gid(team)
-    "#{Season.current.name}-#{Time.now.getutc.to_i}-#{team.id}"
+  def build_conference
+    conference = Conference.new(conference_params)
+    conference.season_id = current_season.id
+    conference.gid = generate_gid
+    conference
+  end
+
+  def generate_gid
+    "#{Season.current.name}-#{Time.now.getutc.to_i}-#{current_user.id}"
```

Comment:
My main question: If I understand this correctly, this means that _every_ logged_in user can add conferences to the main list now? Because this controller is about the conferences that are imported via the importer, right?